info/available-rules.md

```diff
@@ -98,6 +98,7 @@
 |    6 |     6.1.4 | MULTIPLE_INIT_BLOCKS                   | Checks that classes have only one init block | yes | no | - |
 |    6 |     6.1.5 | USELESS_SUPERTYPE                       | Check: if override function can be removed                                                                                                                                   | yes| - |     |
 |    6 |     6.1.6 | CLASS_SHOULD_NOT_BE_ABSTRACT           | Checks: if abstract class has any abstract method. If not, warns that class should not be abstract<br>Fix: deletes abstract modifier                                         | yes | - | - |
+|    6 |     6.1.7 | NO_CORRESPONDING_PROPERTY              | Checks: if in case of using "implicit backing property" scheme, the name of real and back property are the same                                                              | no | - | - |
```

Comment:
```suggestion
|    6 |     6.1.7 | NO_CORRESPONDING_PROPERTY              | Checks: if in case of using "backing property" scheme, the name of real and back property are the same                                                              | no | - | - |
```
It's not implicit though